samples/net/wpan_serial/overlay-cc2520.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_IEEE802154_CC2520=y
+CONFIG_NET_APP_IEEE802154_DEV_NAME="cc2520"
```

Comment:
Following https://github.com/zephyrproject-rtos/zephyr/pull/9391, these should be renamed: CONFIG_NET_APP_* => CONFIG_NET_CONFIG_* (not all NET_APP_* of course, only those related to config).